introduction.md

```diff
@@ -0,0 +1,6 @@
+git clone https://github.com/username/dojo_rules.git
```

Comment:
echo "On my way to becoming a GitHub Master. I love Ruby and JavaScript." > introduction.md
git commit -am "Added Code School Path"
git push